ui/pages/confirm-import-token/confirm-import-token.js

```diff
@@ -0,0 +1,134 @@
+import React, { useContext, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import {
+  ASSET_ROUTE,
+  IMPORT_TOKEN_ROUTE,
+} from '../../helpers/constants/routes';
+import Button from '../../components/ui/button';
+import Identicon from '../../components/ui/identicon';
+import TokenBalance from '../../components/ui/token-balance';
+import { I18nContext } from '../../contexts/i18n';
+import { MetaMetricsContext } from '../../contexts/metametrics';
+import { getMostRecentOverviewPage } from '../../ducks/history/history';
+import { addTokens, clearPendingTokens } from '../../store/actions';
+
+const getTokenName = (name, symbol) => {
+  return typeof name === 'undefined' ? symbol : `${name} (${symbol})`;
+};
+
+const ConfirmImportToken = () => {
+  const metricsEvent = useContext(MetaMetricsContext);
+  const t = useContext(I18nContext);
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const mostRecentOverviewPage = useSelector((state) =>
+    getMostRecentOverviewPage(state),
+  );
+  const pendingTokens = useSelector((state) => state.metamask.pendingTokens);
+
+  const tokenAddedEvent = (pendingToken) => {
```

Comment:
I actually had this same thought to prepend this with `track`. I went with conforming to existing patterns. Thanks for bringing this up! :) 

I think it might be sufficient to use `trackTokenAdded`, but I don't think I know enough context of the app to propose this at this time. I can go with `trackTokenAddedEvent` in the meantime.